cfme/intelligence/reports/widgets/__init__.py

```diff
@@ -187,7 +187,9 @@ def correct_tree_menu_item_selected(self):
         # The bug is currently targeted only to 5.10, but we have the
         # problem on 5.11 as well, so we probably need to use
         # forced_streams.
-        if BZ(1667064, forced_streams=['5.11']).blocks:
+        #
+        # The bug is currently deferred, so normal BZ(*).blocks doesn't block.
```

Comment:
It should, that seems like an issue to me. Deferred should be considered a blocking state by default, testers shouldn't have to include this check.

Reply:
OK.

Reply:
This is done now. The `.blocks`now returns `True` when BZ is deferred.

Reply:
Can we change these calls to `.blocks` now?

Reply:
@john-dupuy Oh!

Reply:
Done. Thanks!